src/uct/api/uct.h

```diff
@@ -1133,6 +1156,12 @@ struct uct_iface_params {
      * +-------------------+
      */
     size_t                                       am_align_offset;
+
+    /**
+     * UCT @ref uct_iface_feature "features" that are used for interface
+     * initialization.
+     */
+    uint64_t                                     features;
```

Comment:
@shamisp 
Would it be better to have a "white-list" of needed features, or a "black-list" of not-needed ones?
When a feature is requested, it might not be deterministic by UCT (depends on TL support), though when it is explicitly excluded  ("black-list") - it is guaranteed to not exist.

Having said that, IMO using "white-list" leads to a cleaner code & simpler logic.

Reply:
Is this essentially request for a minimal set of features ? what a TL does not support specific feature ? Will it be removed from the list ?

Reply:
would be good to mention the default behavior. I think by default iface will support `all`, right?

Reply:
yes, by default support all

Reply:
but it is not clear from the API - not mentioned in the docs

Reply:
@gleon99 can u pls add clarification in separate PR?